src/main/java/edu/cornell/kfs/coa/document/CuSubObjectCodeMaintainableImpl.java

```diff
@@ -0,0 +1,30 @@
+package edu.cornell.kfs.coa.document;
+
+import java.util.List;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.kuali.kfs.krad.maintenance.MaintenanceLock;
+import org.kuali.kfs.krad.maintenance.MaintenanceUtils;
+import org.kuali.kfs.sys.document.FinancialSystemMaintainable;
+import org.springframework.cache.Cache;
+
+public class CuSubObjectCodeMaintainableImpl extends FinancialSystemMaintainable {
+    
+    private static final Logger LOG = LogManager.getLogger();
+    
+    @Override
+    public List<MaintenanceLock> generateMaintenanceLocks() {
+        List<MaintenanceLock> maintenanceLocks = super.generateMaintenanceLocks();
+        
+        Cache cache = MaintenanceUtils.getBlockingCache();
+        if (LOG.isDebugEnabled()) {
+            LOG.debug("generateMaintenanceLocks, clear all blocking cache ");
+        }
+        cache.clear();
```

Comment:
You should also add this clear-all-entries logic to the Account Reversion maintainable.

Reply:
OK, added code to AccountReversionMaintainableImpl